content/soarian/dstu2.md

```diff
@@ -53,6 +53,15 @@ For example:
 Where `https://fhir-myrecord.sandboxcerner.com/dstu2/abcd1234/` is the the `:serviceRootURL`, `Condition` is the 
 `:resource`, and `patientId=4342008` is the `:parameters`.
 
+### Secure Sandbox
+
+The secure sandbox instance can be used for testing an application with [authorization](#authorization). The service root URL for the sandbox instance is:
+
+`https://fhir-myrecord.sandboxcerner.com/dstu2/0e885770-571b-4c0c-b30f-21df9a058d0d`
```

Comment:
this looks like the same URL as above - does Soarian have an open and a secure sandbox? seems that there's no open, so I don't think the h3 for Secure Sandbox makes sense.

Reply:
1.The First Link under secure sandbox section indicates the service root url for secure sandbox. This follows the general format specified under the “Service root URL” section which is why we decided to hyperlink the aforementioned section under “secure sandbox”
2.The Second link indicates an example of the URL for a given recourse call.
Therefore the 2 links present under secure sandbox need to be present.

Reply:
Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d